app/service/data_svc.py

```diff
@@ -379,6 +412,10 @@ def __init__(self, dao):
                     for name, info in executors.items():
                         for e in name.split(','):
                             encoded_test = b64encode(info['command'].strip().encode('utf-8'))
+                            fact_relationships = await self._relationships_from_yaml(ab)
+                            if (ab['platforms'][pl][name]).get('requires', None):
```

Comment:
why just requires here and not consequences?

Reply:
Good call